app/assets/stylesheets/custom.css.scss

```diff
@@ -48,23 +54,109 @@ a:hover, a:active {
     color: $dark-clam-tan;
 }
 
+.btn {
+    padding: 5px;
+    padding-right: 10px;
+    padding-left: 10px;
+}
+
 #sign-in {
     color: $white;
 }
 
 #sign-out {
     color: $white;
+    padding: 5px;
+    padding-right: 10px;
+    padding-left: 10px;
+}
+
+#sign-out:hover {
+    background-color: $lighter-crab-red;
 }
 
 #create-button {
     color: $white;
 }
 
+#cart {
+    color: $white;
+    padding: 5px;
+    padding-right: 10px;
+    padding-left: 10px;
+}
+
+#cart:hover {
+    background-color: $ocean-green;
+}
+
+#create-button {
+    color: $white;
+}
+
+#release {
+    color: $white;
+}
+
+.navbar-default {
+  background-color: $parchment;
+  border-color: $parchment;
+  margin: 5px;
+  margin-bottom: 20px;
+}
+
+.btn-success {
+    background-color: $dark-ocean-green;
+    border-color: $dark-ocean-green;
+    margin-top: 10px;
+    margin: 5px;
+}
+
+.btn-success:hover {
+    background-color: $ocean-green;
+    border-color: $ocean-green;
+}
+
+.btn-danger {
+    background-color: $light-crab-red;
+    border-color: $light-crab-red;
+    margin-top: 10px;
+    margin: 5px;
+}
+
+.btn-danger:hover {
+    background-color: $lighter-crab-red;
+    border-color: $lighter-crab-red;
+}
+
+.polaroid {
+    position: relative;
+}
+
+.polaroid img {
+    border: 10px solid #fff;
+    border-bottom: 45px solid #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces